brand/admin.py

```diff
@@ -314,3 +316,15 @@ def changelist_view(self, request, extra_context=None):
 @admin.register(BrandSuggestion)
 class BrandSuggestionsAdmin(admin.ModelAdmin):
     list_display = ["short_name", "submitter_name", "submitter_email"]
+
+
+@admin.register(EmbraceCampaign)
+class EmbraceCampaignAdmin(admin.ModelAdmin):
+    """
+    This EmbraceCampaignAdmin will add EmbraceCampaign model data into
+    Admin interface
+    """
+
+    model = EmbraceCampaign
+
+    list_display = ["name", "description"]
```

Comment:
You'll probably want to display id here, for the developers to see. That's what they'll be using when configuring new campaigns